federation-js/src/composition/validate/sdl/__tests__/matchingUnions.test.ts

```diff
@@ -87,6 +87,16 @@ describe('MatchingUnions', () => {
     expect(errors[0]).toMatchInlineSnapshot(`
       Object {
         "code": "VALUE_TYPE_UNION_TYPES_MISMATCH",
+        "locations": Array [
```

Comment:
I think this is fine as is, but tossing around the idea of having the location point to specific types on the diff of these unions.

In this case, that would mean the location points to the `Error` type, but I'm not sure we can identify whether the `Error` is extraneous in one service vs. missing in another.

I'm leaning towards keeping it as is, but flagging the thought in case anything interesting comes to mind.